server/gc/src/test/java/org/apache/accumulo/gc/GarbageCollectWriteAheadLogsTest.java

```diff
@@ -250,7 +265,10 @@ public void replicationDelaysFileCollection() throws Exception {
 
     GCStatus status = new GCStatus(null, null, null, new GcCycleStats());
 
+    tserverSet.scanServers();
+    EasyMock.expectLastCall();
```

Comment:
That's a really nice (and simple) improvement to the test to make sure we call scanServers to ensure it is updated before we expect the call to getCurrentServers.